notebooks/geocode_snap_ATRs.ipynb

```diff
@@ -279,21 +319,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 2",
+   "display_name": "Python 3",
```

Comment:
Looks like you're using python 3 instead of python 2.  That might be causing the problems you were talking about?